### DISCUSSION TITLE
Disable suggest changes features

### DISCUSSION THREAD

**Discussion by cdmcdermott** [adityatelange/hugo-PaperMod#389] [Q&A] [answered]
Does the suggest changes feature require a copy of my site (prior to being statically generated by hugo) to be stored in a separate github repo ?

Not sure I like the idea of that, so it is possible to disable these feature globally?

**Comment by adityatelange** [comment, marked as answer]
Please look for this Papermod's wiki section.
This isn't enabled by default!

**Reply by cdmcdermott** [reply]
ah ok, many thanks
